gematria/datasets/pipelines/compile_modules_lib_test.py

```diff
@@ -98,6 +98,38 @@ def test_annotate_bbs(self):
 
     self.assertLen(annotated_blocks, 1)
 
+  def test_get_vocab(self):
+    get_vocab_function = compile_modules_lib.GetVocab()
+    get_vocab_function.setup()
+
+    vocab = list(
+        get_vocab_function.process('4829d38b44246c8b54246848c1fb034829d04839c3')
+    )
+
+    self.assertLen(vocab, 17)
+    self.assertContainsSubset(
+        vocab,
+        [
+            '_MEMORY_',
+            'RAX',
+            '_IMMEDIATE_',
+            'RBX',
+            '_D_',
+            '_ADDRESS_',
+            'EFLAGS',
+            'EAX',
+            'MOV',
+            'EDX',
+            'RSP',
+            '_NO_REGISTER_',
+            'SAR',
+            'CMP',
+            'RDX',
+            'SUB',
+            '_DISPLACEMENT_',
+        ],
+    )
```

Comment:
The expected list already contains 17 elements, i.e. it covers all tokens from the basic blocks.

Then, this is equivalent to:
```py
self.assertCountEqual(vocab, [...the list of expected tokens...])
```
but `assertCountEqual` will produce slightly better error messages on failure (it should print the actual list even if the number of items doesn't match).